Contents/Code/__init__.py

```diff
@@ -121,9 +127,9 @@ def MainMenu():
         oc.add(DirectoryObject(thumb=R('icon_tvshows_hd.jpg'),
                                key=Callback(allrecordings, title="All Recordings", url=Dict['private_ip']),
                                title="Recent Recordings"))
-        oc.add(DirectoryObject(thumb=R('icon_scheduled_hd.jpg'),
-                               key=Callback(scheduled, title="Scheduled Recordings"),
-                               title="Scheduled Recordings"))
+        # oc.add(DirectoryObject(thumb=R('icon_scheduled_hd.jpg'),
```

Comment:
I did not fix the scheduled recordings bit, and have commented it out for the time being